godtools/ViewControllers/Tract/TractViewControllerContent.swift

```diff
@@ -116,4 +123,40 @@ extension TractViewController {
         snapshotView?.removeFromSuperview()
     }
     
+    private func showErrorMessage() {
+        let alert = UIAlertController(title: "error".localized,
+                                      message: "tract_loading_error_message".localized,
+                                      preferredStyle: .alert)
+        
+        let actionYes = UIAlertAction(title: "yes".localized,
+                                      style: .default,
+                                      handler: { action in
+                                        self.redownloadResources()
+        })
+        
+        let actionNo = UIAlertAction(title: "no".localized,
+                                      style: .cancel,
+                                      handler: { action in
+                                        self.disableResource()
+        })
+        
+        alert.addAction(actionYes)
+        alert.addAction(actionNo)
+        present(alert, animated: true, completion: nil)
+    }
+    
+    private func redownloadResources() {
+        DownloadedResourceManager().delete(resource!)
```

Comment:
Just in question, I was checking and get lost with this method and the following, is in some place moving back to Home? Because I can't see it, and it should be nice to move the user back to Home.